nixos/modules/services/mail/sympa.nix

```diff
@@ -416,14 +413,14 @@ in
 
       "d  /run/sympa                   0755 ${user} ${group} - -"
     ]
-    ++ (flip concatMap fqdns (fqdn: [
+    ++ (lib.flip lib.concatMap fqdns (fqdn: [
       "d  ${dataDir}/etc/${fqdn}       0700 ${user} ${group} - -"
       "d  ${dataDir}/list_data/${fqdn} 0700 ${user} ${group} - -"
     ]))
-    #++ (flip mapAttrsToList enabledFiles (k: v:
+    #++ (lib.flip lib.mapAttrsToList enabledFiles (k: v:
```

Comment:
Heh

Reply:
surely i could leave it out, but i always changed the comments as well (in the first part even a markdown example)